tests/api.c

```diff
@@ -27757,7 +27889,283 @@ static int verify_cb(int ok, X509_STORE_CTX *ctx)
 }
 #endif
 
+static void test_wolfSSL_X509_Name_canon(void)
+{
+#if defined(OPENSSL_ALL) && !defined(NO_CERTS) && \
+    !defined(NO_FILESYSTEM) && !defined(NO_SHA) && \
+     defined(WOLFSSL_CERT_GEN) && \
+    (defined(WOLFSSL_CERT_REQ) || defined(OLFSSL_CERT_EXT))
```

Comment:
"OLFSSL_CERT_EXT" should be "WOLFSSL_CERT_EXT"

Reply:
Thank you for pointing it out.